packages/backend/src/core/assets/ArbitrumEBVUpdater.ts

```diff
@@ -99,6 +99,12 @@ export class ArbitrumEBVUpdater implements AssetUpdater {
         timestamp: timestamp.toNumber(),
         minTimestamp: this.minTimestamp.toNumber(),
       })
+      await this.reportStatusRepository.add({
```

Comment:
Why are we polluting our DB with useless information?

Reply:
This way next time backend starts with the same configHash it will not try to update those timestamps, without it we will be skipping every process reload. Maybe it should be handled differently, but I am not sure yet how